inventory.adoc

```diff
@@ -4,7 +4,47 @@
 Production:: https://inventory.izettle.com
 
 ### Description
-The Inventory Service keeps track of the stock level of products. In the API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
+The Inventory Service keeps track of the stock level for variants within products. The API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
```

Comment:
perhaps clarify: "The service automatically decreases the stock when purchases are made with the iZettles POS application."